README.md

```diff
@@ -1,34 +1,49 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
-## Getting Started
-
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-```
-
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
-
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
-
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+<p align="center">
+   <a href="https://hackclub.app">
+      <img src="https://avatars.githubusercontent.com/u/84053203?s=200&v=4" height="96">
+      <h3 align="center">Hack as a Service</h3>
+   </a>
+      <p align="center">📦 The app platform for @hackclub students.</p>
+    
+</p>
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+
+## Documentation
+For more concise details on how to use Hack as a Service, check out our [documentation](https://haas.hackclub.com/docs/)
+
+## Contributing
+
+How can I contribute?
+
+1. Clone the repo.
+   ```sh
+   git clone https://github.com/hack-as-a-service/frontend
+    ```
+2. Change directory.
+   ``` 
+   cd frontend
+   ```
+3. Install dependencies. 
+   ```
+   yarn install
+   ```
+   
```

Comment:
Please make the note that the API server has to be setup (which can be done so by following its readme)

Reply:
I will link the API repo then